src/repository/SensorEventRepository.ts

```diff
@@ -8,7 +8,8 @@ export class SensorEventRepository {
     sensor_spec?: string,
     from_date?: number,
     to_date?: number,
-    limit?: number
+    limit?: number,
+    is_device?:boolean
```

Comment:
Please remove this - this is a standardized API that should not be changed. See my comment below for more details.